libs/eo/certificates/src/lib/certificate-details/eo-certificate-details.component.ts

```diff
@@ -0,0 +1,142 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+import { ActivatedRoute, Router, RouterModule } from '@angular/router';
+import { EoStackComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { eoCertificatesRoutePath } from '@energinet-datahub/eo/shared/utilities';
+import { LetModule } from '@rx-angular/template';
+import { map, tap } from 'rxjs';
+import { EoCertificatesStore } from '../eo-certificates.store';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [
+    MatCardModule,
+    EoStackComponent,
+    CommonModule,
+    RouterModule,
+    LetModule,
+  ],
+  selector: 'eo-certificates',
+  standalone: true,
+  styles: [
+    `
+      :host {
+        display: grid;
+        grid-template-columns: auto 279px;
+        grid-gap: var(--watt-space-m);
+        max-width: 1040px; // Magic UX number
+      }
+
+      .grid-table {
+        display: grid;
+        grid-template-columns: minmax(auto, 200px) auto;
+        gap: 1rem;
+      }
+
+      .space-between {
+        width: 100%;
+        display: inline-flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      .link {
+        text-decoration: none;
+      }
+    `,
+  ],
+  template: `
+    <eo-stack size="M" *rxLet="certificate$; let cert">
+      <mat-card>
+        <eo-stack size="M">
+          <h4><b>Static Data</b></h4>
+          <div class="grid-table">
+            <b>Energy</b>
+            <div>{{ cert?.quantity?.toLocaleString() }} Wh</div>
+            <b>Start time</b>
+            <div>
+              {{ cert?.dateFrom | date: 'HH:mm dd-MMM-y z' | uppercase }}
+            </div>
+            <b>Start time</b>
+            <div>
+              {{ cert?.dateTo | date: 'HH:mm dd-MMM-y z' | uppercase }}
+            </div>
+            <b>GSRN</b>
+            <div>{{ cert?.gsrn }}</div>
+            <b>Certificate ID</b>
+            <div>{{ cert?.id }}</div>
+          </div>
+        </eo-stack>
+      </mat-card>
+      <mat-card>
+        <div class="space-between">
+          <eo-stack size="M">
+            <h4><b>Technology</b></h4>
+            <div class="grid-table">
+              <b>Technology Code</b>
+              <div>{{ cert?.techCode }}</div>
+              <b>Fuel Code</b>
+              <div>{{ cert?.fuelCode }}</div>
+            </div>
+          </eo-stack>
+          <img
+            alt="Windmill"
+            src="/assets/images/certificates/windmill.png"
+            style="height: 79px;"
+          />
+        </div>
+      </mat-card>
+      <h4>
+        <a class="link" routerLink="/${eoCertificatesRoutePath}">
+          << Back to Certificates
+        </a>
+      </h4>
+    </eo-stack>
+    <eo-stack size="M">
+      <mat-card>
+        <eo-stack size="M">
+          <h4><b>Bidding Zone</b></h4>
+          <p><b>DK1</b></p>
+          <img
```

Comment:
Maybe NgOptimizedImage is something to look into?

Reply:
See previous comment :D